### PR TITLE
[Analytics] Adds footnote about API token and key.

### PR DESCRIPTION
**Pull request by angelampcosta**
### Summary

Adds footnote about API token and key.

### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.

### CODE REVIEW

File: src/content/docs/analytics/graphql-api/tutorials/querying-access-login-events.mdx [hunk 1]
```diff
@@ -8,7 +8,7 @@ In this example, we are going to use the GraphQL Analytics API to retrieve logs
 
 The following API call will request logs for a single Access login event and output the requested fields. The authentication request is identified by its **Ray ID**, which you can obtain from the `403` Forbidden page shown to the user.
 
-You will need to insert your API credentials in `<EMAIL>` and `<API_KEY>` and substitute your own values for the following variables:
+You will need to insert your API credentials in `<EMAIL>` and `<API_TOKEN>`[^1] and substitute your own values for the following variables:
```
Comment (victor-perov) [inline src/content/docs/analytics/graphql-api/tutorials/querying-access-login-events.mdx]: the snippet below does not use EMAIL. You don't need to provide email if your authorisation method is an API Token

File: src/content/docs/analytics/graphql-api/tutorials/querying-firewall-events.mdx [hunk 2]
```diff
@@ -6,7 +6,7 @@ title: Querying Firewall Events with GraphQL
 
 In this example, we are going to use the GraphQL Analytics API to query for Firewall Events over a specified time period.
 
-The following API call will request Firewall Events over a one hour period, and output the requested fields. Be sure to replace `<CLOUDFLARE_ZONE_ID>`, `<EMAIL>`, and `<API_KEY>` with your zone tag and API credentials, and adjust the `datetime_geg` and `datetime_leq` values to your liking.
+The following API call will request Firewall Events over a one hour period, and output the requested fields. Be sure to replace `<CLOUDFLARE_ZONE_ID>`, `<EMAIL>`, and `<API_TOKEN>`[^1] with your zone tag and API credentials, and adjust the `datetime_geg` and `datetime_leq` values to your liking.
```
Comment (victor-perov) [inline src/content/docs/analytics/graphql-api/tutorials/querying-firewall-events.mdx]: We should remove `<EMAIL>` and rename CLOUDFLARE_ZONE_ID into CLOUDFLARE_ZONE_TAG for clarity

File: src/content/docs/analytics/graphql-api/tutorials/querying-magic-firewall-ids-samples.mdx [hunk 3]
```diff
@@ -6,7 +6,7 @@ title: Querying Magic Firewall Intrusion Detection System (IDS) samples with Gra
 
 In this example, we are going to use the GraphQL Analytics API to query for IDS samples over a specified time period.
 
-The following API call will request IDS samples over a one hour period, and output the requested fields. Be sure to replace `<CLOUDFLARE_ACCOUNT_ID>`, `<EMAIL>`, and `<API_KEY>` with your account tag and API credentials, and adjust the `datetime_geg` and `datetime_leq` values to your liking.
+The following API call will request IDS samples over a one hour period, and output the requested fields. Be sure to replace `<CLOUDFLARE_ACCOUNT_ID>`, `<EMAIL>`, and `<API_TOKEN>`[^1] with your account tag and API credentials, and adjust the `datetime_geg` and `datetime_leq` values to your liking.
```
Comment (victor-perov) [inline src/content/docs/analytics/graphql-api/tutorials/querying-magic-firewall-ids-samples.mdx]: same with email and id suffix

File: src/content/docs/analytics/graphql-api/tutorials/querying-magic-firewall-samples.mdx [hunk 4]
```diff
@@ -6,7 +6,7 @@ title: Querying Magic Firewall Samples with GraphQL
 
 In this example, we are going to use the GraphQL Analytics API to query for Magic Firewall Samples over a specified time period.
 
-The following API call will request Magic Firewall Samples over a one hour period, and output the requested fields. Be sure to replace `<CLOUDFLARE_ACCOUNT_ID>`, `<EMAIL>`, and `<API_KEY>` with your zone tag and API credentials, and adjust the `datetime_geg` and `datetime_leq` values to your liking.
+The following API call will request Magic Firewall Samples over a one hour period, and output the requested fields. Be sure to replace `<CLOUDFLARE_ACCOUNT_ID>`, `<EMAIL>`, and `<API_TOKEN>`[^1] with your zone tag and API credentials, and adjust the `datetime_geg` and `datetime_leq` values to your liking.
```
Comment (victor-perov) [inline src/content/docs/analytics/graphql-api/tutorials/querying-magic-firewall-samples.mdx]: same as above

File: src/content/docs/analytics/graphql-api/tutorials/querying-workers-metrics.mdx [hunk 5]
```diff
@@ -8,7 +8,7 @@ products:
 
 In this example, we are going to use the GraphQL Analytics API to query for Workers Metrics over a specified time period. We can query up to one month of data for dates up to three months ago.
 
-The following API call will request a Worker script's metrics over a one day period, and output the requested fields. Be sure to replace `<CLOUDFLARE_ACCOUNT_ID>`, `<EMAIL>`, and `<API_KEY>` with your API credentials, and adjust the `datetimeStart`, `datetimeEnd`, and `scriptName` variables as needed.
+The following API call will request a Worker script's metrics over a one day period, and output the requested fields. Be sure to replace `<CLOUDFLARE_ACCOUNT_ID>`, `<EMAIL>`, and `<API_TOKEN>`[^1] with your API credentials, and adjust the `datetimeStart`, `datetimeEnd`, and `scriptName` variables as needed.
```
Comment (victor-perov) [inline src/content/docs/analytics/graphql-api/tutorials/querying-workers-metrics.mdx]: same as above

File: src/content/partials/magic-transit/graphql/query-magic-transit-bandwidth-graphql.mdx [hunk 6]
```diff
@@ -7,7 +7,7 @@ import { Markdown } from "~/components";
 
 In this example, you are going to use the GraphQL Analytics API to query {props.productName} ingress tunnel traffic over a specified time period.
 
-The following API call will request {props.productName} ingress tunnel traffic over a one-hour period and output the requested fields. Be sure to replace `<CLOUDFLARE_ACCOUNT_ID>` with your account ID, `<EMAIL>`, `<API_KEY>`, and `<API_TOKEN>` with your API credentials, and adjust the `datetime_geq` and `datetime_leq` values as needed.
+The following API call will request {props.productName} ingress tunnel traffic over a one-hour period and output the requested fields. Be sure to replace `<CLOUDFLARE_ACCOUNT_ID>` with your account ID, `<EMAIL>`, `<API_KEY>`[^1], and `<API_TOKEN>`[^2] with your API credentials, and adjust the `datetime_geq` and `datetime_leq` values as needed.
```
Comment (victor-perov) [inline src/content/partials/magic-transit/graphql/query-magic-transit-bandwidth-graphql.mdx]: We should clarify here that we need API_KEY and EMAIL only for legacy way of querying. When `API_TOKEN` is more preferable.

File: src/content/partials/magic-transit/graphql/query-magic-transit-health-checks.mdx [hunk 7]
```diff
@@ -7,7 +7,7 @@ import { Markdown } from "~/components";
 
 In this example, you are going to use the GraphQL Analytics API to query {props.productName} health check results which are aggregated from individual health checks carried out by Cloudflare servers to Generic Routing Encapsulation (GRE) tunnels you have set up to work with {props.productName} during the onboarding process. You can query up to one week of data for dates up to three months ago.
 
-The following API call will request a particular account's tunnel health checks over a one day period for a particular Cloudflare data center, and outputs the requested fields. Be sure to replace `<CLOUDFLARE_ACCOUNT_ID>`, `<EMAIL>`, and `<API_KEY>` with your API credentials, and adjust the `datetimeStart`, `datetimeEnd` variables as needed.
+The following API call will request a particular account's tunnel health checks over a one day period for a particular Cloudflare data center, and outputs the requested fields. Be sure to replace `<CLOUDFLARE_ACCOUNT_ID>`, `<EMAIL>`, and `<API_TOKEN>`[^1] with your API credentials, and adjust the `datetimeStart`, `datetimeEnd` variables as needed.
```
Comment (victor-perov) [inline src/content/partials/magic-transit/graphql/query-magic-transit-health-checks.mdx]: same for EMAIL, ID

File: src/content/docs/analytics/graphql-api/tutorials/querying-access-login-events.mdx (hunk 1 above)
Comment (victor-perov) [reply src/content/docs/analytics/graphql-api/tutorials/querying-access-login-events.mdx]: Also, lets use CLOUDFLARE_ACCOUNT_TAG here too

**Review by victor-perov — CHANGES_REQUESTED** [review]
Thanks, @angelampcosta!

Since we are amending these snippets, I think we should clear from leftovers of the legacy querying method.

**Review by victor-perov — APPROVED** [review]
LGTM

File: src/content/docs/analytics/graphql-api/tutorials/end-customer-analytics.mdx [hunk 8]
```diff
@@ -8,7 +8,7 @@ title: Querying HTTP events by hostname with GraphQL
 
 In this example, we are going to use the GraphQL Analytics API to query aggregated metrics about HTTP events by hostname over a specific period of time.
 
-The following API call will request the number of visits and edge response bytes for the custom hostname `hostname.example.com` over a four day period. Be sure to replace `CLOUDFLARE_ZONE_ID` AND `API_TOKEN` with your zone ID and API credentials, and adjust the `datetime_geq` and `datetime_leq` values as needed.
+The following API call will request the number of visits and edge response bytes for the custom hostname `hostname.example.com` over a four day period. Be sure to replace `CLOUDFLARE_ZONE_ID` and `API_TOKEN`[^1] with your zone ID and API credentials, and adjust the `datetime_geq` and `datetime_leq` values as needed.
```
Comment (RebeccaTamachiro) [inline src/content/docs/analytics/graphql-api/tutorials/end-customer-analytics.mdx]: Disregard if not applicable, but this caught my eye: Should this be updated to `CLOUDFLARE_ZONE_TAG`, as it was in the [querying-firewall-events.mdx below](https://github.com/cloudflare/cloudflare-docs/pull/18036/files#diff-bce12b4ed184980dc7df8a9652a11c2081683f76b2965b75c86e276a993df6c3R9)?

**Review by RebeccaTamachiro — APPROVED** [review]

**Pull request merged**